src/main/assets/js/data-layer.ts

```diff
@@ -1,2 +1,14 @@
 const dataLayer = window.dataLayer || [];
 dataLayer.push({ event: 'Site language', language: document.documentElement.lang });
+
+const errorList: HTMLUListElement | null = document.querySelector('.govuk-error-summary__list');
+if (errorList) {
+  const listElements = errorList.getElementsByTagName('li');
+  for (let i = 0; i < listElements.length; i++) {
+    dataLayer.push({
+      event: 'Error shown',
+      errormsg: listElements.item(i)?.firstElementChild?.textContent,
+    });
+    console.log('error shown', listElements.item(i)?.firstElementChild?.textContent);
```

Comment:
Is this just for testing? Do we need it?

Reply:
ah, yeah, was meant to remove it :|

Reply:
done